pretyping/pretyping.mli

```diff
@@ -44,7 +44,8 @@ type typing_constraint =
   | OfType of types (** A term of the expected type *)
   | WithoutTypeConstraint (** A term of unknown expected type *)
 
-type inference_hook = env -> evar_map -> Evar.t -> evar_map * constr
+(* Some = did suceeed *)
```

Comment:
this comment is both useless and mistyped

Reply:
Thanks for the review @gares ; comment amended, I hope it is better now.